README.md

```diff
@@ -9,7 +9,7 @@ bun is a new:
 - JavaScript/TypeScript/JSX transpiler
 - JavaScript & CSS bundler
 - Task runner for package.json scripts
-- npm-compatible package manager
+- Npm-compatible package manager
```

Comment:
I think "npm" is used as a proper noun in here which itself starts with a lowercase "n"

Reply:
i dont think this is a correct change.
while "npn" is a name of a "thing" and therefore a noun it is still in small letters. 

"npm" is the correct way to write it. If you look on the official website it is written with 3 small letters everywhere. And i think thoose guys that make the thing know how to name it. :)

Reply:
That's exactly what I'm saying? I don't know how I came across, but I'm sure that those guys that make the thing know how to name it too! 🤠 As "npm".